container_linux.go

```diff
@@ -249,6 +257,124 @@ func (c *linuxContainer) NotifyOOM() (<-chan struct{}, error) {
 	return notifyOnOOM(c.cgroupManager.GetPaths())
 }
 
+// XXX debug support, remove when debugging done.
```

Comment:
@vishh 
Since we're still debugging, it makes a lot easier to add additional command line options when invoking CRIU through the environment.  Otherwise, we need to edit and recompile source for every little change in the options.  I will remove it soon.